docs/en/ingest-management/commands.asciidoc

```diff
@@ -36,19 +36,29 @@ invoking the wrong binary.
 [[elastic-agent-diagnostics-command]]
 == elastic-agent diagnostics
 
-Gather diagnostics information from the {agent} and applications it's running.
+Gather diagnostics information from the {agent} and component/unit it's running.
 
-If no options are specified, this command displays version numbers and application metadata.
+If no options are specified, this command displays version numbers and component/unit metadata.
 
-If `collect` is specified, it produces an archive containing application metadata, configuration information, the policy, local logs, and metrics data (if the buffer is enabled).
+If `collect` is specified, it produces an archive containing:
+
+* version.txt - version information
+* pre-config.yaml - pre-configuration before variable substitution
+* variables.yaml - current variable contexts from providers
+* computed-config.yaml - configuration after variable substitution
+* components.yaml - expected computed components model from the computed-config.yaml
+* state.yaml - current state information of all running components
+* goroutine.txt - goroutine dump
+* heap.txt - memory allocation of live objects
+* allocs.txt - sampling past memory allocations
+* threadcreate.txt - traces led to creation of new OS threads
+* block.txt - stack traces that led to blocking on synchronization primitives
+* mutex.txt - stack traces of holders of contended mutexes
```

Comment:
I would put a note that this information is only for the Elastic Agent. There is also a `components` directory that includes diagnostic information from each running component. Contents of those directories are defined by the inputs.